src/System.ServiceModel.Duplex/pkg/System.ServiceModel.Duplex.pkgproj

```diff
@@ -6,7 +6,7 @@
       <SupportedFramework>net45;win8;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\src\System.ServiceModel.Duplex.builds" />
```

Comment:
You retargeted the src assembly but didn't harvest the old asset: lib/netstandard1.3